test/test_utils/__init__.py

```diff
@@ -115,6 +115,11 @@ def get_inference_server_type(image_uri):
         return "mms"
     return "ts"
 
+def get_build_context():
+    return os.getenv("BUILD_CONTEXT")
+
+def get_eks_role():
+    return os.getenv("EKS_TEST_ROLE")
```

Comment:
Why does this need to be an env variable?

It would be better to place this function in test_utils/eks.py instead.

Reply:
Running the test assumes that an EKS cluster is in active state. We have added rbac rules in cluster for `EKS_TEST_ROLE ` to give minimum permission required to run the test job. Hence, the kubeconfig must be updated to use such role.

This is not mandatory for a customer, i.e. the cx can use the cluster creator role to run test in which case, the `--authenticator-role-arn` is not necessary.

Will place the function in `test_utils/eks.py`